docs/modules/ROOT/pages/how-tos/worklows/git-submodules.adoc

```diff
@@ -0,0 +1,90 @@
+= Git Submodule pattern
+
+== How this workflow pattern works:
+
+1. Create a "downstream" repo.
+2. Track your upstream git repo with a git submodule reference.
+3. Onboard to {ProductName}, use link:https://github.com/renovatebot/renovate[Renovate] to keep your Git submodule references up to date.
+4. In your downstream repo, apply patches in your Containerfile if you need to carry downstream patches.
+
+== Implementing this solution pattern
+=== Create a downstream repo
+
+A **downstream repo** is a self-owned copy of the upstream repository whose build, test, release lifecycle you want to control completely.
+
+The downstream repo will:
+
+* be used to hold the git submodule reference to the upstream repository.
+* contain a Containerfile that references and pulls in the git submodule(s) content.
+* become a Component of our {ProductName} application.
```

Comment:
Maybe just state something like it will be onboarded to {ProductName}? There isn't _necessarily_ only a single Application. The fact that the component belongs to an application is likely irrelevant here.

Reply:
changed to "onboard onto {ProductName}"